lib/components/Snackbar/__tests__/snackbar.test.tsx

```diff
@@ -0,0 +1,109 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { Snackbar } from "..";
+
+describe("Snackbar", () => {
+    it("renders with default props", () => {
+        const { container } = render(
+            <Snackbar
+                message="Test message"
+                isOpen={true}
+                onClose={jest.fn()}
+            />,
+        );
+        expect(screen.getByText("Test message")).toBeInTheDocument();
```

Comment:
Maybe it's better to create constants for strings if you are commonly using them?